substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -33,9 +33,9 @@
 //!	Query the default genesis config from the provided `runtime.wasm` and use it in the chain
 //! spec. Tool can also store runtime's default genesis config in given file:
 //!	```text
-//! chain-spec-builder create -r runtime.wasm default /dev/stdout
+//! chain-spec-builder --chain_spec_path /dev/stdout create -r runtime.wasm default
```

Comment:
```suggestion
//! chain-spec-builder create -r runtime.wasm default /dev/stdout
```

Reply:
This API feels confusing to me. I call `chain-spec-builder create`, and following the docs put a file path on the end, which I assume should set the path of the chain-spec output (I am calling chain-spec-builder create after all), but the output is not a chain spec but something different?

Reply:
Well, I think description clearly says it stores config not chain-spec.
Writing genesis config is convenient, optional operation.

```
$ ./chain-spec-builder create default --help
Gets the default genesis config for the runtime and uses it in ChainSpec. Please note that default genesis config may not be valid. For some runtimes initial values should be added there (e.g. session keys, babe epoch)

Usage: chain-spec-builder4 create --runtime-wasm-path <RUNTIME_WASM_PATH> default [DEFAULT_CONFIG_PATH]

Arguments:
  [DEFAULT_CONFIG_PATH]  If provided stores the default genesis config json file at given path (in addition to chain-spec)

Options:
  -h, --help  Print help
  ```

The chain-spec path is given by main `-c` option.

Reply:
If you feel confused with this example feel free to change `/dev/stdout` to `default_genesis_config.json`, maybe it will be more clear.